lib/private/intercept-exit-callbacks.js

```diff
@@ -245,7 +246,7 @@ module.exports = function interceptExitCallbacks (callbacks, _cache, hash, machi
             // But it may or may not have already had its stack combined with
             // the stacks in the environment.  If not, we'll do it now.
             if (!value.stackModified) {
-              value.stack = value.message + '\n' + getCombinedErrorStack(value.stack.split('\n').slice(1), machine._configuredEnvironment.debug.stack);
+              value.stack = value.message + '\n' + getCombinedErrorStack(stripStackMessage(value).split('\n'), machine._configuredEnvironment.debug.stack);
```

Comment:
Can we include the `value.name + ': '+` prefix here? (see comment above on line 810 for explanation)